plugins/infoblox/icon_infoblox/actions/search_by_ip/action.py

```diff
@@ -1,25 +1,22 @@
-import komand
-from .schema import SearchByIpInput, SearchByIpOutput
+import insightconnect_plugin_runtime
+from .schema import SearchByIpInput, SearchByIpOutput, Input, Output, Component
 
 # Custom imports below
 
 
-class SearchByIp(komand.Action):
+class SearchByIp(insightconnect_plugin_runtime.Action):
     def __init__(self):
         super(self.__class__, self).__init__(
             name="search_by_ip",
-            description="Search hosts by IP address",
+            description=Component.DESCRIPTION,
             input=SearchByIpInput(),
             output=SearchByIpOutput(),
         )
 
     def run(self, params={}):
-        ip = params.get("ip")
+        ip = params.get(Input.IP)
         objects = self.connection.infoblox_connection.search_by_ip(ip)
         result = []
         for o in objects:
             result.extend(o["objects"])
```

Comment:
This could be refactored to use list comprehension with `.get()` method for obtaining objects from object.